test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -12860,6 +12860,24 @@ def add_config(self, conf={}, hup=True):
 
         return self.apply_config(conf, hup)
 
+    def unset_mom_config(self, name, hup=True):
```

Comment:
Ah! I see, my bad I didn't read the DD for this change as I was just reviewing PTL changes. Anyways Thanks for ticket id, will do needful.